src/AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -9,7 +9,7 @@ public partial class TestSuiteConstraints
         public bool SupportsNativePubSub { get; } = false;
         public bool SupportsNativeDeferral { get; } = false;
         public bool SupportsOutbox { get; } = false;
-        public IConfigureEndpointTestExecution CreateTransportConfiguration() => new ConfigureEndpointAzureStorageQueueTransport();
+        public IConfigureEndpointTestExecution CreateTransportConfiguration() => new ConfigureEndpointAcceptanceTestingTransport(SupportsNativePubSub, SupportsNativeDeferral);
```

Comment:
Where can I find any info on the workings of this transport?

Reply:
Its a copy and paste of the LearningTransport

https://docs.particular.net/transports/learning/?version=core_7

But I guess as they start to diverge we need to start documenting the differences?

Reply:
Why is `SupportsCrossQueueTransactions` false?